database/seeds/BodySeeder.php

```diff
@@ -48,36 +29,324 @@ public function run()
             'name'          =>  'commission-european',
         ]);
 
+	Address::create([ //TODO: Validate address
+            'country_id'    =>  202,
+            'street'        =>  'Campus de Elviña',
+            'zipcode'       =>  '15071',
+            'city'          =>  'A Coruna',
+        ]);
+
     	Body::create([
-            'type_id'       =>  2,
+            'type_id'       =>  1,
             'address_id'    =>  1,
-    		'name'			=>	'AEGEE-Europe',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'AEU' //No clue if this is actually correct.
+    	    'name'	    =>	'AEGEE-A Coruna',
+            'email'         =>  'aegee.coruna@gmail.com',
+	    'phone'	    =>  '',
+            'legacy_key'    =>  'TMP' //TODO
```

Comment:
Same as for the phone number:
Based on https://github.com/AEGEE/oms-neo-core/blob/develop/database/migrations/2017_04_28_151956_new_database_schema.php#L101 this is nullable (optional)